resources/addon.go

```diff
@@ -14,17 +14,18 @@ type Addon struct {
 
 // Fetch fetches addon having the given addonID.
 func (addon *Addon) Fetch(addonID string, queryParams map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
-	url := fmt.Sprintf("%s/%s", constants.ADDON_URL, addonID)
+	url := fmt.Sprintf("/%s%s/%s", constants.VERSION_V1, constants.ADDON_URL, addonID)
```

Comment:
it looks ok compared to the previous URL but there should be one more '/' after v1 can we recheck this?